### PR TITLE
Move validate to first in Jenkins test stages

### PR DESCRIPTION
**Pull request by JakeQuilty**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -13,6 +13,14 @@ pipeline {
   }
 
   stages {
+    stage('Validate') {
+      parallel {
```
Comment (andytinkham) [inline Jenkinsfile]: Why are you using parallel for just a single step? Are you planning for other specific validate steps in the future, or just planning for the potential for more validation (like with the static analysis stuff I'm working on?)

**Comment by sgnn7** [comment]
Given low availability overlaps between @JakeQuilty and rest of the team, nits have been fixed/updated.

File: Jenkinsfile (hunk 1 above)
Comment (sgnn7) [reply Jenkinsfile]: Fixed

**Comment by izgeri** [comment]
@JakeQuilty - please see Hugh's feedback above - it's a good point, it's important to give some reasoning about what motivates a change in your PR / commit messages.

**Pull request merged**